recipe/build.sh

```diff
@@ -22,9 +13,10 @@ fi
              --system-libs \
              --no-qt-gui \
              --no-system-libarchive \
+             --no-system-jsoncpp \
```

Comment:
Yes, but that's for the build step after bootstrap phase is done. We can do the bootstrapping parallely as well.